pkg/query/paging/README.md

```diff
@@ -101,26 +102,38 @@ func ValidateAndApplyPagingRules(model PagingSettingsInterface, request *Request
 ValidateAndApplyPagingRules performs a validation of the original request and adds correct the correct values \(defaults\) if needed
 
 <a name="Response"></a>
-## type [Response](<https://github.com/greenbone/opensight-golang-libraries/blob/main/pkg/query/paging/response.go#L6-L10>)
+## type [Response](<https://github.com/greenbone/opensight-golang-libraries/blob/main/pkg/query/paging/response.go#L8-L13>)
 
 Response represents a response object containing information about pagination and total count of records.
 
-- PageIndex: The index of the page \(starting from 0\).
-- PageSize: The number of records per page.
+- PageIndex: The index of the page \(starting from 0\). This is required.
+- PageSize: The number of records per page. This is required.
+- TotalDisplayableResults: The total number of results that can be displayed. This is required.
```

Comment:
This is not needed as this is the length of the result slice

Reply:
I updated the doc string to explain why this is needed. 
The context is for the 10k restrictions on [opensearch](https://opensearch.org/docs/latest/search-plugins/searching-data/paginate/#:~:text=Querying%20for%20pages%20deep%20in,This%20can%20cause%20inconsistent%20pagination.)  
`TotalResults` is only returned when there are restrictions on the size of data that the query can retrieve. It returns the total count as it is in the database. This helps the front-end display the limit to the customer, and add tips to use the filter